app/src/main/java/com/sopt/now/presentation/auth/login/LoginActivity.kt

```diff
@@ -0,0 +1,45 @@
+package com.sopt.now.presentation.auth.login
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.databinding.ActivityLoginBinding
+import com.sopt.now.presentation.data.model.dto.RequestLoginDto
+import com.sopt.now.presentation.auth.signup.SignupActivity
+
+class LoginActivity : AppCompatActivity() {
+    private val binding by lazy { ActivityLoginBinding.inflate(layoutInflater) }
+    private val viewModel by viewModels<LoginViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        clickButtonListener()
+    }
+
+    private fun clickButtonListener() {
+        // 로그인 버튼 클릭
+        binding.btnLogin.setOnClickListener {
+            viewModel.login(getLoginRequestDto())
+        }
+        // 회원가입 버튼 클릭
+        binding.btnSignup.setOnClickListener {
+            Intent(this@LoginActivity, SignupActivity::class.java).let {
+                startActivity(it)
+            }
+        }
+    }
+
+    private fun getLoginRequestDto(): RequestLoginDto {
+        val id = binding.edtLoginId.text.toString()
+        val password = binding.edtLoginPassword.text.toString()
+        return RequestLoginDto(
+            authenticationId = id,
+            password = password,
+        )
+    }
+    companion object {
+        var memberId = "1"
+    }
```

Comment:
```suggestion
    companion object {
        const val MEMBER_ID = "1"
    }
```
상수화하려면 const val로 바꿔줘야 할 것 같아요! 그리고 저는 보통 변하지 않는 값을 강조하기 위해 대문자로 표시해줍니다!